packages/grant-explorer/src/features/discovery/LandingPage.tsx

```diff
@@ -75,11 +82,17 @@ const LandingPage = () => {
         action={<ViewAllLink to="#">View all</ViewAllLink>}
       >
         <div className="grid md:grid-cols-3 gap-x-6">
-          {activeRounds.data?.map((round) => (
-            <div key={round.id}>
-              <RoundCard round={round} />
-            </div>
-          ))}
+          {activeRounds.data
+            ?.filter(
+              (round) =>
+                round?.projects?.length !== undefined &&
+                round.projects.length > 0
+            )
```

Comment:
This can be removed by moving the filter to the hook.
Also, it's sufficient with `round.projects?.length > 0`